.github/workflows/unit-tests.yaml

```diff
@@ -0,0 +1,38 @@
+name: Tests
+on:
+  push:
+    tags:
+    - '*'
+    branches:
+    - master
+    - release-*
+  pull_request:
+    types:
+    - opened
+    - synchronize
+jobs:
+  ui:
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.49
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Create combined mono repo lock file
+      uses: ./.github/actions/create-concatenated-ui-monorepo-lock
+
+    - name: Cache UI dependencies
+      uses: ./.github/actions/cache-ui-dependencies
+
+    - name: Fetch UI deps
+      run: make -C ui deps
+
+    - name: UI Unit Tests
+      run: make ui-test
+
+    - name: Publish Test Report
+      uses: mikepenz/action-junit-report@v3
```

Comment:
Stupid question. I see that this action runs and reports how many tests are successful. But where do I see the test results? E.g. test names, their status (passed/failed) and any stdout+stderr from them (that's captured in junit output).

Reply:
This looks extremely bare-bones and pretty much on the border of being completely unusable.

Would you be so kind to try find some GitHub Action or a different tool that displays results in more actionable way, as a backlog/follow-up item, or at least create a follow-up JIRA?